try {
	var num = 10;
	console.log(stu);
	console.log(num);
} catch (error) {
	console.log("Error");
} finally {
	console.log(num);
}
